browse/templates/abs/extra_services.html

```diff
@@ -56,14 +56,10 @@
       {%- endif -%}
       {%- if format == 'src' and current-%}
   <li><a href="{{url_for('src.src', arxiv_id=requested_id)}}" class="abs-button download-eprint">TeX Source</a></li>
-      {%- elif format.startswith('ps') -%}
-  <li><a href="{{url_for('dissemination.ps', arxiv_id=requested_id)}}" class="abs-button download-ps">PostScript{% if format.endswith('(400)') %} (400 dpi){% elif format.endswith('(600)') %} (600 dpi){% elif format.endswith('(cm)') %} (Type I cm){% elif format.endswith('(CM)') %} (Type I CM){% endif %}</a></li>
-      {%- elif format == 'dvi' -%}
-  <li><a href="{{url_for('dissemination.dvi', arxiv_id=requested_id)}}" class="abs-button download-dvi">{{ format.upper() }}</a></li>
       {%- elif format == 'html' -%}
   <li><a href="{{url_for('dissemination.html', arxiv_id=requested_id)}}" accesskey="f" class="abs-button download-html">{{ format.upper() }}</a></li>
-      {%- elif format == 'nops' -%}
```

Comment:
As far as I can tell from ARXIVCE-1159 this PR isn't suppose to remove the 'nops' format from the list.

Reply:
Mark tells me this PR is intended to remove the 'nops' download link.